components/chart/base/MultiBarBaseChart.tsx

```
@@ -0,0 +1,101 @@
+                                                 scaleUnit,
+                                                 reverse,
+                                                 noMargin,
+                                             }: MultiBarBaseChartProps) {
+    const [ height, setHeight ] = useState(0);
+    const [ width, setWidth ] = useState(0);
+
+    const maxValue = useMemo(
+        () =>
+            Math.max(...values.map((value, index) => {
+                return Math.max(...value.map((value2, index2) => {
+                    return value2 + valuesOffset?.[index]?.[index2]! }))
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)